install.sh

```diff
@@ -75,6 +75,8 @@ install_nvm_from_git() {
       exit 1
     }
   else
+    echo "=> Cleanup old nvm dir '$INSTALL_DIR'"
+    command rm -rf $INSTALL_DIR/* $INSTALL_DIR/.*
```

Comment:
we absoLUTELY do not want to do this - this would delete previously installed node versions, along with all their global modules.